examples/epd1in54_no_graphics/Cargo.toml

```diff
@@ -8,5 +8,5 @@ edition = "2018"
 
 epd-waveshare = { path = "../../", default-features = false, features = ["epd1in54"]}
 
-linux-embedded-hal = "0.2.2"
+linux-embedded-hal = "0.5.2"
```

Comment:
copy-paste mistake most likely. this is an example without the embedded-graphics lib so nothing should be changed here ;-)